### ISSUE TITLE
Add some more mail automation tools

### ISSUE THREAD

**Issue by meghpatel** [codezoned/ScriptsDump#115]

**Comment by ionicc** [comment]
Yes, we were looking into that. Would you like to take this issue up @meghpatel ?

**Comment by srivastava9** [comment]
i am taking this

**Comment by meghpatel** [comment]
> Yes, we were looking into that. Would you like to take this issue up @meghpatel?

Yes, I've a basic Mail sending tool implemented in Python

**Comment by meghpatel** [comment]
> i am taking this

I saw your PR of Mail Automation. I'll add my stuff once it's accepted
